tests/preferences/widgets/conftest.py

```diff
@@ -75,6 +75,8 @@ def simple_adjustment(request, faker):
     """Return three random numbers in an ascending order."""
```

Comment:
Should read  "Return a ``SimpleAdjustment``". Which of cause *is* three random numbers but this seems more consistent with the fixture name and actually returned class.